lib/composer/package/loader/project_attributes_loader.rb

```diff
@@ -0,0 +1,56 @@
+module Composer

```

Comment:
I tried moving the classes to Gitlab::Composer namespace but I can't get them to load their parent classes. Alternatively I can prefix the classes with Gitlab_______ . Let me know if this works for you.